actions/marketplace.js

```diff
@@ -22,6 +22,10 @@ export function fetchPlugins() {
 
             return {plugins};
         } catch (error) {
+            // If the marketplace server is unreachable, try to get the local plugins only.
```

Comment:
Why is this logic client side?

Reply:
This was a tricky one because given that now it would include local plugins in the result we needed to distinguish when there was an actual error connecting to the remote marketplace and when it was just returning the local plugins because it was disabled. 

It came down to either returning a new bit of information and changing the already published API endpoint, or adding the new bit to get local plugins only when the remote makretplace was enabled but unreacheable (which would be the exception). We went with the latter approach